theia-electron/package.json

```diff
@@ -49,6 +52,8 @@
         "@theia/navigator": "next",
         "@theia/outline-view": "next",
         "@theia/output": "next",
+        "@theia/plugin-ext": "next",
```

Comment:
we should clean up Theia extensions, i.e. all extensions replaced by built-in should go as well as typescript. @vince-fugnitto Could you help with it?

Actually package.json could be minimal like:
```json
{
  dependencies: {
     "@theia/plugin-ext-vscode": "next" ,
     "@theia/json": "next" ,
     "@theia/json": "preferences" 
  }
}
```
The rest will come transitively.

Reply:
As much as I like small configs, when it comes to Theia extensions it is not really clear what to strip out and what to keep. Being explicit as the advantage of being very transparent.

Reply:
ok, but deprecated should be removed